vertical-pod-autoscaler/pkg/recommender/model/cluster.go

```diff
@@ -90,14 +94,20 @@ type PodState struct {
 	Phase apiv1.PodPhase
 }
 
+// IsActive returns true if Pod is in active phase.
```

Comment:
Looks like a duplicate from spec_client.go - do we need both?

Reply:
They work on different types. (BasicPodSpec and PodStatus) and  I can try and merge into one if it's worth it.

Reply:
Removed.